package.json

```diff
@@ -25,7 +28,14 @@
   ],
   "author": "Gaidarenko Oleg <markelog@gmail.com>",
   "license": "MIT",
+  "dependencies": {
+    "@stylistic/eslint-plugin-js": "^2.9.0"
+  },
+  "peerDependencies": {
+    "eslint": ">=9.0.0"
+  },
   "devDependencies": {
-    "eslint": "^8.5.0"
+    "eslint": "^9.12.0",
+    "jquery": "latest"
```

Comment:
I'd prefer to provide a concrete version here, even without the `^` range. This is because code style changes are not breaking and may happen even in patch releases, breaking the build.